src/destreamer.ts

```diff
@@ -1,7 +1,8 @@
 import { sleep, parseVideoUrls, checkRequirements, makeUniqueTitle, ffmpegTimemarkToChunk } from './utils';
+import { setProcessEvents } from './destreamerEvents';
```

Comment:
Can you make it `Events.ts` instead of `destreamerEvents.ts` to stay consistent?